scripts/data-load.sh

```diff
@@ -0,0 +1,59 @@
+#!/bin/bash
+
+echo "register: $1"
+echo "phase: $2"
+
+echo ""
+echo "Downloading $2 field-records.json"
+rm -f field-records.json
+curl -sS https://field.$2.openregister.org/records.json > field-records.json
+
+echo ""
+[[ -e ../$1-data ]] || echo "cloning $1-data repo"
+[[ -e ../$1-data ]] || (cd .. && git clone git@github.com:openregister/$1-data.git)
+echo "../$1-data repo: checkout master && pull"
+cd ../$1-data && git checkout master && git fetch origin && git pull
```

Comment:
was this meant to be the `git fetch && git merge`? I'm not sure why you'd do a `pull` after a `fetch`?